indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -153,3 +154,12 @@ def _process(self):
         booking_details['occurrences'] = occurrences
         booking_details['date_range'] = date_range
         return jsonify(booking_details)
+
+
+class RHBookingEventData(RHBookingDetails):
+    def _process(self):
+        if self.booking.event is None:
+            raise NotFound
```

Comment:
hmm... 404 vs response indicating that there is no booking. i guess with 404 the error handling in the JS is a bit trickier

Reply:
Well, we will never handle it in JS, since we only send the request in the first place if we know there is an event.

Reply:
ah even better, hadn't noticed that the request is conditional. then i agree that 404 is a good option